apps/src/p5lab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -21,6 +21,8 @@ import style from './animation-picker-body.module.scss';
 import AnimationUploadButton from './AnimationUploadButton.jsx';
 
 const MAX_SEARCH_RESULTS = 40;
+const LEVEL_COSTUMES = 'level_costumes';
```

Comment:
My first thought was to see where this `constants` file would be located since `animationLibraryApi.js` is located in `src/assetManagement` whereas `AnimationPickerBody` is in `src/p5lab/AnimationPicker`.  There is a `src/constants.js` file but the constant doesn't seem like it belongs there. So I think I prefer as is. But if you feel strongly, I can add - just lmk where you prefer. Thanks!